assets/js/modules/sign-in-with-google/components/common/AnyoneCanRegisterReadOnly.js

```diff
@@ -59,7 +59,7 @@ export default function AnyoneCanRegisterReadOnly() {
 	return (
 		<div className="googlesitekit-settings-module__fields-group googlesitekit-settings-module__fields-group--read-only">
 			<span>
-				{ __( 'Anyone can register WP setting', 'google-site-kit' ) }
```

Comment:
Again, different from Figma so we need to align with Sigal on the language so that it's consistent in both places.

I feel like we could find a better title still as well. "Users can create new accounts" as a section heading could be misunderstood to be the current state. How about something like "User registration"?

Reply:
"User registration" works for me 👍🏻